### ISSUE TITLE
Update pext annotations

### ISSUE THREAD

**Issue by gtiao** [broadinstitute/gnomad_qc#68]

**Comment by gtiao** [comment]
Requires GTEx v9 update

**Comment by gtiao** [comment]
I asked Francois Aguet about the status of the GTEx update, which we need to be on an updated Gencode version.

As of Feb 24, he said: "We have the V9 VCF now, but the RNA-seq re-processing will be in V10. I don't have an estimate for when that will happen (asap of course, but we're severely bandwidth-limited) -- but next 2-3 months sounds doable. We'll use the latest available GENCODE version."

I asked what happens if that Gencode is more updated than the one we used for gnomAD (i.e., will we have to re-VEP all our gnomAD variants again?).

Francois: "I haven't checked yet, but expect the differences in annotation to be much more minor than in the past, so maybe VEP could be re-run for the subset of changed loci/transcripts?"

tl;dr: still waiting on GTEx to update Gencode annotations, will have to wait to see what version that's on to decide how to handle reconciliation with gnomAD

**Comment by nawatts** [comment]
Replaced by broadinstitute/gnomad_production#132.
